packages/manager/modules/vps/src/dashboard/vps-dashboard.constants.js

```diff
@@ -12,10 +12,19 @@ export const NEW_RANGE_VERSION = '2019v1';
 export const VPS_2014_AUTO_MIGRATION_DATE = '18/07/2020';
 export const VPS_2014_EXPIRY_DATE = '27/07/2020';
 
+export const RTM_WARNING_VALUE = {
+  min: 50,
+  max: 80,
+};
+
+export const GUIDE_RTM = 'https://www.ovh.com/fr/';
```

Comment:
Even though it might only be available for FR now, maybe we could 
```suggestion
export const GUIDE_RTM = {
FR: 'https://www.ovh.com/fr/';
}
```
to be able to easily add subs/regions